src/renate/run/training_job.py

```diff
@@ -140,9 +140,9 @@ def execute_tuning_job(
         backend in defaults.SUPPORTED_BACKEND
     ), f"Backend {backend} is not in {defaults.SUPPORTED_BACKEND}."
     if backend == "local":
-        return _execute_tuning_job_locally(
-            state_url=state_url,
-            next_state_url=next_state_url,
+        return _execute_tuning_and_tuning_job_locally(
```

Comment:
a bit odd and lengthy name

Reply:
I agree, it's a long name but I think it's good to be specific. 
Also, this is not exposed to the user.

Reply:
Why is it tuning _and_ tuning job? I don't understand this.